makefile

```diff
@@ -0,0 +1,8 @@
+install:
+	@echo "[install] Installing dependencies":
+	@npm i -g nodemon
```

Comment:
please, don't use nodejs dependencies, because this project uses golang with the main language, this way we avoid creating dependencies with other environments